pkg/components/metadata.go

```diff
@@ -14,12 +14,16 @@
 
 package components
 
+import (
+	"github.com/kinvolk/lokomotive/pkg/k8sutil"
+)
+
 // Metadata is a struct which represents basic information about the component.
 // It may contain information like name, version, dependencies, namespace, source etc.
 type Metadata struct {
-	Name      string
-	Namespace string
-	Helm      HelmMetadata
+	Name             string
+	ReleaseNamespace k8sutil.Namespace
```

Comment:
`Release` is helm-specific term. I'd just name this field `Namespace`.
```suggestion
	ReleaseNamespace k8sutil.Namespace
```